### PR TITLE
an/SKY-32: Create profile edit view

### PR DESCRIPTION
**Pull request by anurmamedov**
Profile edit form

### CODE REVIEW

**Comment by anurmamedov** [comment]
*email section removed from profile edit
*bio name changed to status in profile edit
*gender section added in profile edit , not functioning yet.

**Comment by anurmamedov** [comment]
i can't see any deployment gives 404 error, not found

File: templates/profile-edit.html [hunk 1]
```diff
@@ -2,94 +2,80 @@
...
                 </div>
             </div>
-        </div>
+        </form>
     </div>
     <script>
         document.addEventListener("DOMContentLoaded",() => {
-            document.getElementById('imagefile').addEventListener('change', (event) => {
-                image = document.getElementById('imagefile')
+            document.getElementById('profileimagefile').addEventListener('change', (event) => {
```
Comment (Aymammet) [inline templates/profile-edit.html:77]: what about if we follow name conventions as "xxx-yyyy"
as example : "profile-image", "image-file"

File: templates/profile-edit.html [hunk 2]
```diff
@@ -2,94 +2,80 @@
...
-                                </div>
-                            </div>
-                            <!-- Save changes button-->
+
                             <div class="">
                                 <a href="{% url 'profile' user.id %}" class="btn btn-primary mx-1">
                                     Profile
                                 </a>
-                                <button class="btn btn-primary" type="button">Save changes</button>
+                                <button class="btn btn-primary" type="submit">Save changes</button>
```
Comment (Aymammet) [inline templates/profile-edit.html:67]: Use "Save" instead "Save changes"

File: templates/profile-edit.html [hunk 3]
```diff
@@ -2,94 +2,80 @@
...
+                                </div>
+                            </div>
+                            <label class="small mb-1" for="inputGender">Gender</label>
+                            <div class="mb-3 col-4">
+                                <div class="form-outline mb-4 input-group flex-nowrap">
+                                    <span class="input-group-text fa-solid fa-venus-mars fs-5" id="addon-wrapping"></span>
+                                    <input type="text" class="form-control" name="gender" id="checkboxgender" placeholder="Your gender"  value="{{ form.gender.value }}">
                                 </div>
                             </div>
-                            <!-- Form Row        -->
```
Comment (Aymammet) [inline templates/profile-edit.html:55]: do we need this one?

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Comment by anurmamedov** [comment]
understand, can you review my last commit please

File: templates/profile-edit.html [hunk 4]
```diff
@@ -81,22 +76,23 @@
...
             </div>
-        </div>
+        </form>
     </div>
     <script>
         document.addEventListener("DOMContentLoaded",() => {
-            document.getElementById('imagefile').addEventListener('change', (event) => {
-                image = document.getElementById('imagefile')
+            document.getElementById('image-file').addEventListener('change', (event) => {
+                image = document.getElementById('image-file')
```
Comment (Aymammet) [inline templates/profile-edit.html:95]: Arent theese changes made in your another branch?
 [https://github.com/Aymammet/sky/pull/70]

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/profile-edit.html (hunk 4 above)
Comment (anurmamedov) [reply templates/profile-edit.html:95]: yes, the rest was done in another branch because it included post-edit file

**Review by anurmamedov — COMMENTED** [review]
please check last review.

File: templates/profile-edit.html (hunk 2 above)
Comment (anurmamedov) [reply templates/profile-edit.html]: changed in to Save
image

File: templates/profile-edit.html (hunk 1 above)
Comment (anurmamedov) [reply templates/profile-edit.html]: changed to profile-image and profile-file

File: templates/profile-edit.html (hunk 3 above)
Comment (anurmamedov) [reply templates/profile-edit.html:55]: all comments are removed

**Review by anurmamedov — COMMENTED** [review]
all comments been followed by requests.

**Review by Aymammet — APPROVED** [review]

**Pull request merged**